src/card.ts

```diff
@@ -101,7 +101,13 @@ export interface Card {
   newPanFromCardToken?:string;
   translatePinFromCardToken?: string;
   activationActions?: ActivationActions;
+  barcode?: string;
   bulkIssuanceToken?: string;
+  chipCvvNumber?:string;
+  contactlessExemptionCounter?:bigint;
+  contactlessExemptionTotalAmount?:number;
+  cvvNumber?:string;
```

Comment:
These last four... again, the style of a space after the `:` and before the type...